cjs/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Can you please make a new folder for JS code checked by TS compiler, instead of modifing the pure cjs?

Reply:
I'm not really modifying it, just adding an independent step on top of it, but sure I can duplicate the folder. Just pushed it